docs/gallery/howto/autogen/parallel_wf.py

```diff
@@ -0,0 +1,278 @@
+"""
```

Comment:
Note that RTD caches the other file somehow, therefore I renamed it. I don't fully understand this.